store/iavl/store.go

```diff
@@ -43,16 +43,16 @@ type Store struct {
 // LoadStore returns an IAVL Store as a CommitKVStore. Internally, it will load the
 // store's version (id) from the provided DB. An error is returned if the version
 // fails to load, or if called with a positive version on an empty tree.
-func LoadStore(db dbm.DB, logger log.Logger, key types.StoreKey, id types.CommitID, lazyLoading bool, cacheSize int, disableFastNode bool, metrics metrics.StoreMetrics) (types.CommitKVStore, error) {
```

Comment:
since we still have a mix of legacy keys and new key types should we add lazyloading back?

Reply:
`lazyLoading` was used to avoid loading all available versions previously in the archive nodes.
Now the iavl is not loading all versions even though have a mix of legacy and new format.
Instead, it is checking directly from storage.